source/_components/n26.markdown

```diff
@@ -0,0 +1,49 @@
+---
+layout: page
+title: "N26"
+description: "Instructions on how to integrate N26 component within Home Assistant."
+date: 2019-03-11 20:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: n26.png
+ha_category:
+  - Sensor
+  - Switch
+ha_release: 0.90
+featured: false
+ha_iot_class: "Local Polling"
+redirect_from:
```

Comment:
because the component is new, this is not necessary

Reply:
removed